src/content/docs/mobile-monitoring/new-relic-mobile-flutter/monitor-your-flutter-application.mdx

```diff
@@ -30,7 +31,16 @@ Our New Relic Flutter agent monitors your Flutter mobile app and provides deep i
 
 ## (Recommended) Guided installation [#guided-install]
 
-To install the Flutter agent, follow our [guided install](https://onenr.io/0qQagg4AGR1), located directly in the UI.
+To install the Flutter agent, follow the guided install:
+1. Go to **[one.newrelic.com](https://one.newrelic.com/) > Integrations & Agents**.
```

Comment:
I'd add the <DNT> and </DNT> tags in the path.

Reply:
Thanks for this callout!